tests/test_driver.py

```diff
@@ -226,6 +226,21 @@ def test_exclude_accounts(self, **_):
         )
         self.assertFalse("accounts" in credit_report)
 
+    @patch.multiple(
+        mintapi.Mint,
+        _get_api_key_header=DEFAULT,
+        _load_mint_credit_url=DEFAULT,
+        _get_credit_reports=DEFAULT,
+        get_credit_inquiries=DEFAULT,
+        get_credit_accounts=DEFAULT,
+    )
+    def test_exclude_utilization(self, **_):
+        mint = mintapi.Mint()
+        credit_report = mint.get_credit_report(
+            limit=2, details=True, exclude_utilization=True
+        )
+        self.assertFalse("utilization" in credit_report)
```

Comment:
Nice work! Maybe this is how the other tests work and I missed it, but if the key changed from "utilization", this test would still keep on passing under all scenarios. I'd prefer that we added a line to each exclusion test that verifies the key *does* exist without the exclusion option specified, and then that it goes away when you do exclude it. What do you think?

Reply:
@mrooney good point.  I like it!  I'll do that for this test and the others as well.